src/main/java/com/google/cloud/genomics/utils/grpc/VariantUtils.java

```diff
@@ -60,6 +168,65 @@ public Integer apply(String string) {
       });
 
   /**
+   * Determines whether the variant is filtered according to the FILTER field,
+   * using the notation required by the VCF 4.2 standard. This means that the
+   * FILTER field of the variant must either be empty, or contain only "PASS"
+   * or ".".
+   */
+  public static final Predicate<Variant> IS_FILTERED = new Predicate<Variant>() {
+    @Override
+    public boolean apply(Variant variant) {
+      return variant.getFilterCount() != 0 && !((variant.getFilterCount() == 1)
+          && (variant.getFilter(0).equalsIgnoreCase(PASSES_FILTERS) ||
+          variant.getFilter(0).equalsIgnoreCase(MISSING_VALUE)));
+    }
+  };
+
+  /**
+   * Determines whether a variant has one and only one alternate allele.
+   */
+  public static final Predicate<Variant> IS_BIALLELIC = Predicates.and(HAS_ALTERNATE,
+      new Predicate<Variant>() {
+        @Override
+        public boolean apply(Variant variant) {
+          return variant.getAlternateBasesList().size() == 1;
+        }
+      }
+  );
+
+  /**
+   * Determines whether a variant is a gVCF record.
+   */
+  public static final Predicate<Variant> IS_GVCF = new Predicate<Variant>() {
+    @Override
+    public boolean apply(Variant variant) {
+      return variant.getAlternateBasesCount() == 1 && variant.getAlternateBases(0).equals("<*>");
```

Comment:
I'm not sure the above criteria is all-encompassing.  Isn't the following also a valid gVCF record with two alternate counts of `A,<NON_REF>`?

```
20  10001298    .   T   A,<NON_REF> 1404.77 .   DP=41;MLEAC=2,0;MLEAF=1.00,0.00;MQ=171.56;MQ0=0 GT:AD:DP:GQ:PL:SB   1/1:0,41,0:41:99:1438,123,0,1438,123,1438:0,0,0,0
```